homeassistant/components/cast/media_player.py

```diff
@@ -159,7 +200,9 @@ def internal_callback(name):
         ))
 
     _LOGGER.debug("Starting internal pychromecast discovery.")
-    listener, browser = pychromecast.start_discovery(internal_callback)
+    listener, browser = pychromecast.start_discovery(internal_add_callback,
```

Comment:
In retrospect, it was such a bad idea to return tuples from methods. Can never be changed easily.